build.gradle

```diff
@@ -66,50 +51,58 @@ task checkstyle(type: Checkstyle) {
     source = sourceSets.main.allJava + sourceSets.test.allJava
 }
 
-task gem(type: JRubyExec, dependsOn: ["gemspec", "classpath"]) {
-    jrubyArgs "-rrubygems/gem_runner", "-eGem::GemRunner.new.run(ARGV)", "build"
-    script "${project.name}.gemspec"
-    doLast { ant.move(file: "${project.name}-${project.version}.gem", todir: "pkg") }
+embulkPlugin {
+    mainClass = "org.embulk.parser.jsonpath.JsonpathParserPlugin"
+    category = "parser"
+    type = "jsonpath"
 }
 
-task gemPush(type: JRubyExec, dependsOn: ["gem"]) {
-    jrubyArgs "-rrubygems/gem_runner", "-eGem::GemRunner.new.run(ARGV)", "push"
-    script "pkg/${project.name}-${project.version}.gem"
+publishing {
+    publications {
+        embulkPluginMaven(MavenPublication) {  // Publish it with "publishEmbulkPluginMavenPublicationToMavenRepository".
+            from components.java  // Must be "components.java". The dependency modification works only for it.
+        }
+    }
+    repositories {
+        maven {
+            url = "${project.buildDir}/mavenPublishLocal"
+        }
+    }
 }
 
-task "package"(dependsOn: ["gemspec", "classpath"]) {
-    doLast {
-        println "> Build succeeded."
-        println "> You can run embulk with '-L ${file(".").absolutePath}' argument."
+gem {
+    authors = [ "Hiroyuki Sato","Takuma kanari" ]
+    email = [ "hiroysato@gmail.com","chemtrails.t@gmail.com"]
+    // "description" of the gem is copied from "description" of your Gradle project.
+    summary = "JSON parser with JSONPath plugin for Embulk"
+    homepage = "https://github.com/hiroyuki-sato/embulk-parser-jsonpath"
+    licenses = [ "MIT" ]
+
+    generateRubyCode = false  // Avoiding generate lib/embulk/parser/jsonpath.rb
+    into("lib/embulk/parser/") {
```

Comment:
Do we still need this section?

Reply:
Thank you for your comment. I'll take a look more deeply. I don't understand completely yet.

Reply:
Let's double-check later. I still need this part.
embulk-parser-jsonpath supports guess feature. gradle plugins doesn't support guess yet. So, I copy these codes explicitly.
https://gist.github.com/hiroyuki-sato/3e261ef8eb2d092b3d83ae637b9ffd29#gistcomment-3175408

Reply:
I see.